docs/upgrading_to_coil3.md

```diff
@@ -20,17 +17,11 @@ Coil's Maven coordinates were updated from `io.coil-kt` to `io.coil-kt.coil3` an
 
 The `coil-base` and `coil-compose-base` artifacts were renamed to `coil-core` and `coil-compose-core` respectively to align with the naming conventions used by Coroutines, Ktor, and AndroidX.
 
-## Default Changes
-
-Coil 3.0 has several import default changes:
-
-- Cache headers are no longer respected by default. See [here](network.md) for more info.
-- A file's last write timestamp is no longer added to its cache key by default. This is to avoid reading the disk on the main thread (even for a very short amount of time). This can be renabled with `ImageRequest.Builder.addLastModifiedToFileCacheKey` or `ImageLoader.Builder.addLastModifiedToFileCacheKey`.
-- Output image dimensions are now forced to be less than 4096x4096 to guard against accidental OOMs. This can be configured with `ImageLoader/ImageRequest.Builder.maxBitmapSize`.
-
 ## Network Images
 
-`coil-core` no longer supports loading images from the network by default. [**You must** add a dependency on one of Coil's network artifacts. See here for more info.](network.md). This was changed so consumers could use different networking libraries or avoid a network dependency if their app doesn't need it. 
+**`coil-core` no longer supports loading images from the network by default.** [**You must** add a dependency on one of Coil's network artifacts. See here for more info.](network.md). This was changed so consumers could use different networking libraries or avoid a network dependency if their app doesn't need it.
```

Comment:
Right now, there's no `network.md` checked in, so the link in the docs points to 404. Is this intended?